tests/test_integration/test_partner_resource.py

```diff
@@ -0,0 +1,35 @@
+from tests.test_integration.helpers import create_partner
+from tests.test_integration.test_case import YolaServiceTestCase
+
+
+class TestYolaPartners(YolaServiceTestCase):
+    """Yola: Partner resource"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestYolaPartners, cls).setUpClass()
+        cls.partner = create_partner(cls.service)
+        cls.partner_id = cls.partner['id']
+        # There's no way to really delete the test partners we're creating :(
+
+    def test_can_create_partner(self):
+        self.assertEqual(self.partner['properties']['website'], 'example.com')
```

Comment:
This test is following the pattern of "do the work in the setup, verify it in the test". It's true that all the other tests would fail if we can't create a partner, but these are integration tests so they aren't isolated. Plus I'd still like it verified/documented with an explicit test name.